package.json

```diff
@@ -24,13 +24,13 @@
     "in-publish": "^2.0.0",
     "npm-run-all": "^4.0.2",
     "rimraf": "^2.6.1",
-    "tslint": "^4.5.1",
-    "tslint-config-shopify": "^1.5.0",
-    "typescript": "^2.2.2"
+    "tslint": "^5.8.0",
+    "tslint-config-shopify": "^3.0.1",
+    "typescript": "^2.5.3"
   },
   "dependencies": {
     "@types/lodash": "^4.14.65",
-    "@types/react": "^15.0.21",
+    "@types/react": "^16.0.2",
```

Comment:
why is this in here at all?

Reply:
I too was wondering that.

Reply:
looks like geomery's `getRectForNode`.

Reply:
Ah, looks like it is required because of this type we use in geometry: https://github.com/Shopify/javascript-utilities/blob/dependency-updates/src/geometry.ts#L38